products/fuse/get-started.adoc

```diff
@@ -1,190 +1,134 @@
 :awestruct-layout: product-get-started-fuse
 :awestruct-interpolate: true
 
-
-## Extra section title
-Where to begin?
-
-## Extra section
-
-Welcome! This document details the steps to get you up and running quickly with the Red Hat JBoss Fuse runtime and tooling.
-
-#### Install and configure your development environment
-The following steps will install a local version of JBoss Developer Studio 8.1.0 (the Eclipse Luna edition) along with the Fuse tooling.
-
-. Install the Java SE Development Kit (JDK), we recommend Oracle JDK or OpenJDK. (Must be Java 7 compatible.)
-. Download the JBDS 8.1.0 standalone installer from http://developers.redhat.com/products/devstudio/download/ <http://developers.redhat.com/products/devstudio/download/> or click the link here - https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-standalone_jar.jar <https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-standalone_jar.jar>
+:jbdsis-standalone-installer-download-url: https://developers.redhat.com/download-manager/file/devstudio-integration-stack-9.0.2.GA-standalone-installer.jar
+:fuse-on-karaf-download-url: https://developers.redhat.com/download-manager/file/jboss-fuse-karaf-6.3.0.redhat-187.zip
+:installation-on-eap-link: https://access.redhat.com/documentation/en/red-hat-jboss-fuse/6.3/paged/installation-on-jboss-eap
+
+## Fuse Runtime
+
+=== Prerequisites
+Before you start, make sure you have installed the following software:
+
+* Java SE Development Kit (JDK) version 1.8.x (Java 8) - we recommend Oracle JDK or OpenJDK
+* https://maven.apache.org/download.cgi[Apache Maven]
+
+[[fuse-karaf-install-runtime]]
+=== Install and configure the JBoss Fuse Karaf runtime
+video::183851396[vimeo]
+
+. Download the {fuse-on-karaf-download-url}[JBoss Fuse Karaf] runtime package.
+. Unpack the downloaded `.zip` file to a convenient location on your file system, `_FuseInstall_`.
+. Add a user to the Fuse runtime.
+Open the `_FuseInstall_/etc/users.properties` file in a text editor and look for the line starting with `\#admin=admin...`.
+Delete the `#` character at the start of the line and customize the username, `USERNAME`, and password, `PASSWORD` in the user entry:
++
+----
+USERNAME=PASSWORD,admin,manager,viewer,Operator, Maintainer, Deployer, Auditor, Administrator, SuperUser
+----
++
+Save the `users.properties` file.
+
+=== Set up your development environment
+video::183852576[vimeo]
+
+. Download the {jbdsis-standalone-installer-download-url}[Red Hat JBoss Developer Studio Integration Stack 9.0.2.GA Stand-Alone Installer].
```

Comment:
Just to clarify, this link (and the preceding links) are defined by the following variables, which are set at the top of the get-started.adoc file:

`:jbdsis-standalone-installer-download-url: https://developers.redhat.com/download-manager/file/devstudio-integration-stack-9.0.2.GA-standalone-installer.jar
:fuse-on-karaf-download-url: https://developers.redhat.com/download-manager/file/jboss-fuse-karaf-6.3.0.redhat-187.zip
:installation-on-eap-link: https://access.redhat.com/documentation/en/red-hat-jboss-fuse/6.3/paged/installation-on-jboss-eap`